src/v/cloud_storage/segment_meta_cstore.h

```diff
@@ -880,6 +880,11 @@ class segment_meta_cstore {
 public:
     using const_iterator = segment_meta_materializing_iterator;
 
+    using int64_delta_alg = details::delta_delta<int64_t>;
+    using int64_xor_alg = details::delta_xor;
+    using counter_col_t = segment_meta_column<int64_t, int64_delta_alg>;
+    using gauge_col_t = segment_meta_column<int64_t, int64_xor_alg>;
```

Comment:
I see these are also defined in the cc file. Maybe delete the other ones?

Reply:
the one from cc is used by another class